src/main/resources/application.yml

```diff
@@ -28,5 +26,12 @@ spring:
           validation:com.softserve.messaging.model.UserValidation
 
 
+  jpa:
+    hibernate:
+      ddl-auto: validate
+
+  liquibase:
+    change-log: classpath:changelog.db/changelog-master.xml
```

Comment:
don't name folders using a `.`, it's not a Java package.
![image](https://user-images.githubusercontent.com/23016452/77816759-110f2300-70ce-11ea-86e5-285859d0e727.png)